packages/__docs__/src/index.js

```diff
@@ -30,9 +30,7 @@ import { InstUISettingsProvider } from '@instructure/emotion'
 import '../globals'
 
 createRoot(document.getElementById('app')).render(
-  <React.StrictMode>
```

Comment:
Why did you take StrictMode out?

Reply:
it was only for mistake and forgot to put it back